cmd/performance-profile-creator/cmd/root.go

```diff
@@ -249,13 +253,15 @@ type ProfileCreatorArgs struct {
 	SplitReservedCPUsAcrossNUMA bool   `json:"split-reserved-cpus-across-numa"`
 	DisableHT                   bool   `json:"disable-ht"`
 	RTKernel                    bool   `json:"rt-kernel"`
-	UserLevelNetworking         bool   `json:"user-level-networking"`
+	UserLevelNetworking         *bool  `json:"user-level-networking,omitempty"`
```

Comment:
Could you please clarify in the commit message what's the benefit of not adding this option at all, rather than adding with value false?

Reply:
Sure, I thought is was clear, we don't want to have:
    kind: PerformanceProfile 
    spec:
    ...
    net:
       userLevelNetworking: false
   ...
in the output profile is the user did not pass --user-level-networking to the ppc cmd line.

Reply:
In the commit message:
```    Do not automatically add the net.userLevelNetworking field to the performance profile output
    if the user does not specify the corresponding user-level-networking param.
```

Reply:
I'll add "the benefit is to have a more concise profile and the user being able to understand what fields were 'touched'".

Reply:
done

Reply:
thanks!